backend/app/libs/tasks/get_assets.py

```diff
@@ -34,8 +36,13 @@
 
 
 async def make_treasury(treasury_address: str, chain_id: int) -> Treasury:
+    token_whitelist = retrieve_token_whitelist()
```

Comment:
Nitpicking: I'm ok with this, but let's try keeping `get_assets` module as simple as possible and not concerned with implem details. I would rather put these 3 lines in a separate function in the `tokenlists` module.